packages/website/docs/Features/Functions.mdx

```diff
@@ -78,37 +117,141 @@ As well as mixed distributions:
 
 <SquiggleEditor initialSquiggleString="mx(3, 8, 1 to 10, [0.2, 0.3, 0.5])" />
 
-## Other Functions
+An alias of `mx` is `mixture`
+
+### Validity
+
+Using javascript's variable arguments notation, consider `mx(...dists, weights)`:
+
+- `dists.length == weights.length`
+
+## Addition (horizontal right shift)
+
+<SquiggleEditor initialSquiggleString="dist1 = 1 to 10; dist2 = triangular(1,2,3); dist1 + dist2" />
+
+## Subtraction (horizontal left shift)
+
+<SquiggleEditor initialSquiggleString="dist1 = 1 to 10; dist2 = triangular(1,2,3); dist1 - dist2" />
+
+## Multiplication (??)
+
+<SquiggleEditor initialSquiggleString="dist1 = 1 to 10; dist2 = triangular(1,2,3); dist1 * dist2" />
+
+## Division (???)
```

Comment:
@OAGr see multiplication and division `???`. I don't have a good intuition pump for these (in the way that "horizontal shift" covers addition and subtraction)

Reply:
similar for `^ : dist -> dist -> dist`

Reply:
I don't think we should call it these things in the titles. I wouldn't typically call addition a "right shift".